pkg/manager/rest.go

```diff
@@ -21,9 +24,25 @@ func List[T runtime.Object](ctx context.Context, client *rest.RESTClient, resour
 		Into(obj)
 
 	if err != nil {
+		log.Trace("manager.List",
+			slog.Group("args",
+				"resource", resource,
+				"namespace", namespace,
+				"err", err,
+			),
+		)
```

Comment:
Should this be a `log.Error` instead of Trace?

Also I would not Group everything. I like the idea of grouping the `args` and print `args.resource`, `args.namespace`, but I would not print the error in the args.

And the log message should be meaningful, so not `"manager.List"` but something like "error listing resource".

Reply:
It should be also possible to extract the Group, or use a WithXXX

Reply:
Let's discuss about `Group` / `With` offline.